wsmaster/che-core-api-workspace/src/main/resources/schema/devfile.json

```diff
@@ -145,6 +145,14 @@
                 "examples": [
                   "349d3ad"
                 ]
+              },
+              "sparseCheckoutDir": {
+                "type": "string",
+                "description": "The directory which will be created with sparse checkout.",
+                "examples": [
+                  "core/",
+                  "wsmaster/che-core-api-workspace/"
```

Comment:
may it start with `/`? How git will handle that? Just the same as without `/`?
If using `/` at the beginning leads to some errors - we can prevent it on JSON Schema level by specifying format.

Reply:
Good point. There will be no errors, but behaviour will be different. I think this should be described in docs.